### PR TITLE
Issue: 5018, enable logging to tether.debug

### LINKED ISSUE #5018: Longevity failure:  unable to wait for extra config property - context deadline exceeded
[container-logs.zip](https://github.com/vmware/vic/files/976355/container-logs.zip)

from portlayer:
```
May  4 2017 09:38:41.666Z ERROR Unable to wait for extra config property guestinfo.vice..sessions|6e219874010e4d9311e657cc066a7f78ec75f1d607617daaecbaa2de767d4e18.started: Post https://bart.eng.vmware.com/sdk: context deadline exceeded
May  4 2017 09:38:41.666Z DEBUG [ END ] [github.com/vmware/vic/lib/portlayer/exec.(*containerBase).start:181] [3m1.027904986s] 6e219874010e4d9311e657cc066a7f78ec75f1d607617daaecbaa2de767d4e18
May  4 2017 09:38:41.666Z DEBUG [ END ] [github.com/vmware/vic/lib/portlayer/exec.(*Container).start:336] [3m1.027934322s] 6e219874010e4d9311e657cc066a7f78ec75f1d607617daaecbaa2de767d4e18
May  4 2017 09:38:41.666Z DEBUG [BEGIN] [github.com/vmware/vic/lib/portlayer/exec.(*Container).RefreshFromHandle:319] a674123b1e8dab2ae307dbaa3c0b1115
May  4 2017 09:38:41.666Z DEBUG Current ChangeVersion: 2017-05-04T09:35:53.067278Z
May  4 2017 09:38:41.666Z DEBUG [ END ] [github.com/vmware/vic/lib/portlayer/exec.(*Container).RefreshFromHandle:319] [9.041µs] a674123b1e8dab2ae307dbaa3c0b1115
May  4 2017 09:38:41.666Z DEBUG [ END ] [github.com/vmware/vic/lib/portlayer/exec.Commit:35] [3m1.289145972s] 6e219874010e4d9311e657cc066a7f78ec75f1d607617daaecbaa2de767d4e18
May  4 2017 09:38:41.666Z ERROR CommitHandler error on handle(a674123b1e8dab2ae307dbaa3c0b1115) for 6e219874010e4d9311e657cc066a7f78ec75f1d607617daaecbaa2de767d4e18: &errors.errorString{s:"unable to wait for process launch status: Post https://bart.eng.vmware.com/sdk: context deadline exceeded"}
```

### PR DESCRIPTION
**Pull request by lcastellano**
Fixes #5018, partial fix. Enable container logging to tether.debug to collect more data when bootstrap fails. Also added integration test to validate the content of tether.debug.

### CODE REVIEW

File: tests/test-cases/Group9-VIC-Admin/9-01-VICAdmin-ShowHTML.robot [hunk 1]
```diff
@@ -110,6 +110,13 @@ Get Container Logs
     ${vmName}=  Get VM Display Name  ${container}
     Should Contain  ${output}  ${vmName}/vmware.log
     Should Contain  ${output}  ${vmName}/tether.debug
+    ${rc}  ${output}=  Run And Return Rc and Output  curl -sk %{VIC-ADMIN}/container-logs.tar.gz -b /tmp/cookies-%{VCH-NAME} | (cd /tmp; tar xvzf - ${vmName}/tether.debug)
+    Log  ${output}
+    ${rc}  ${output}=  Run And Return Rc and Output  ls -l /tmp/${vm_name}/tether.debug
```
Comment (chengwang86) [inline tests/test-cases/Group9-VIC-Admin/9-01-VICAdmin-ShowHTML.robot]: ${vm_name} --> ${vmName}

File: tests/test-cases/Group9-VIC-Admin/9-01-VICAdmin-ShowHTML.robot [hunk 2]
```diff
@@ -110,6 +110,13 @@ Get Container Logs
...
+    Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${output}=  Run And Return Rc and Output  grep 'prepping for switch to container filesystem' /tmp/${vmName}/tether.debug
+    Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${output}=  Run And Return Rc and Output  rm -f /tmp/${vmName}/tether.debug
```
Comment (chengwang86) [inline tests/test-cases/Group9-VIC-Admin/9-01-VICAdmin-ShowHTML.robot]: If you don't need to check ${rc} and ${output}, maybe just use `Run  rm -f ...`. This is minor though :)

File: tests/test-cases/Group9-VIC-Admin/9-01-VICAdmin-ShowHTML.robot [hunk 3]
```diff
@@ -110,6 +110,13 @@ Get Container Logs
     ${vmName}=  Get VM Display Name  ${container}
     Should Contain  ${output}  ${vmName}/vmware.log
     Should Contain  ${output}  ${vmName}/tether.debug
+    ${rc}  ${output}=  Run And Return Rc and Output  curl -sk %{VIC-ADMIN}/container-logs.tar.gz -b /tmp/cookies-%{VCH-NAME} | (cd /tmp; tar xvzf - ${vmName}/tether.debug)
```
Comment (hickeng) [inline tests/test-cases/Group9-VIC-Admin/9-01-VICAdmin-ShowHTML.robot]: Why are we downloading the logs a second time? The logs have already been downloaded and extracted on line 107.

File: isos/bootstrap/bootstrap [hunk 4]
```diff
@@ -47,6 +47,9 @@ if mount -t ext4 /dev/disk/by-label/containerfs ${MOUNTPOINT}; then
 
     cp /bin/tether ${MOUNTPOINT}/.tether/tether
 
+    echo 'container file system size before copying libraries: '
```
Comment (hickeng) [inline isos/bootstrap/bootstrap]: Calling this `container file system` is a little misleading - it's the tmpfs specifically for holding the tether, related binaries and files.

**Pull request merged**